docs/getting-started/_index.md

```diff
@@ -4,3 +4,55 @@ description: >
   Installation and running your first gadget.
 weight: 10
 ---
+
+Inspektor Gadget can be used either on a Linux machine or on Kubernetes. It
+supports both ephemeral commands and permanent installation.
+
+![Different running modes](how-install-ig.png)
+
+![Different running modes](how-install-ig2.png)
```

Comment:
how-install-ig2.png is done with Excalidraw. The source is included in the png metadata, so it can be edited back without loss. You can see it with `identify -verbose docs/getting-started/how-install-ig2.png`